_posts/2023-08-29-cartsguru.md

```diff
@@ -0,0 +1,122 @@
+---
+layout: post
+title: "[CVE-2023-39642] Improper neutralization of SQL parameter in Carts Guru | Marketing automation multicanal module for PrestaShop"
+categories: modules
+author:
+- TouchWeb.fr
+- 202 Ecommerce
+- Friends-Of-Presta.org
+meta: "CVE,PrestaShop,cartsguru"
+severity: "critical (9.8)"
+---
+
+In the module "Carts Guru | Marketing automation multicanal" (cartsguru) up to versions 2.4.2 from Carts Guru for PrestaShop, a guest can perform SQL injection in affected versions.
+
+
+## Summary
+
+* **CVE ID**: [CVE-2023-39642](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2023-39642)
+* **Published at**: 2023-08-29
+* **Platform**: PrestaShop
+* **Product**: cartsguru
+* **Impacted release**: <= 2.4.2 [considered to be "truly" fixed on 2.4.3 - see note below]
+* **Product author**: Carts Guru
+* **Weakness**: [CWE-89](https://cwe.mitre.org/data/definitions/89.html)
+* **Severity**: critical (9.8)
+
+## Description
+
+The method `CartsGuruCatalogModuleFrontController::display()` and the ajax script controllers14/catalog.php has sensitive SQL calls that can be executed with a trivial http call and exploited to forge a SQL injection.
+
+Note : The author has deleted from its module the files that have been suffering from critical vulnerabilities for months, BUT did not set them to be "auto-deleted" during upgrades. Therefore, there are likely merchants out there with older versions who have updated their modules thinking they are safe. However, there is nothing safe about this since past upgrades do not auto-delete the implicated files. To ensure everyone has a "safe version", we decided to mark all versions up to 2.4.3 as impacted by this issue.
+
+**WARNING** : This exploit is actively used to deploy webskimmer to massively steal credit cards. 
```

Comment:
**WARNING** : This exploit is actively used to deploy a webskimmer to massively steal credit cards.